pages/about_data_page.py

```diff
@@ -1,12 +1,242 @@
+
+                    **Energy Performance Certificates (EPC) register data about homes**: Property data comes from England and Wales and Scotland's EPC register. The EPC register provides data on building characteristics and energy efficiency measures, including: property address and other location information; property characteristics such as number of rooms, property type and built form. Heating system(s) installed; Energy efficiency ratings. The EPC Register datasets are open-source and accessible to everyone.
+
+                    **Joint MCS-EPC dataset**: The joint MCS-EPC dataset is created by linking the MCS heat pump installations data with EPC data. The linkage is done using property address information. More information in this [GitHub repository](https://github.com/nestauk/asf_daps).
+
+                    The joint MCS-EPC dataset is used to derive the following inputs to the lifetime cost model:
+                    - Property archetypes
+                    - Annual heat demand for property archetypes
+                    - Air source heat pump installation costs for each property archetype and cost decile
+
+                    #### Property archetypes
+                    
+                    The user can select/see results for 8 different housing archetypes: flats; semi-detached & terraced houses and maisonettes; detached houses; and bungalows; with each group split into pre- and post-1950 construction.
+
+                    #### Annual heat demand for property archetypes
+
+                    Annual heat demand is estimated for the above archetypes. To learn more about how annual heat demand is estimated, please visit the [GitHub repository](https://github.com/nestauk/asf_heat_pump_affordability/blob/dev/asf_heat_pump_affordability/notebooks/Archetype_Heat_Demand.py)
+                    The annual heat demand for property archetypes uses data up to QX 2024 and is shown below:
+                    """)
+        heat_demand = data_getters.get_property_heat_demand()
+        st.dataframe(heat_demand)
+
+        st.markdown("""
+                    #### Air source heat pump installation costs for each property archetype and decile
+                    Costs of installing Air Source Heat Pumps are estimated for the above housing archetypes and at different cost deciles. Costs are adjusted for inflation against a chosen base year.
+
+                    The air source heat pump installation costs for each property archetype and cost decile uses data up to QX 2024 and is shown below. You can select the cost decile to view the corresponding installation costs in £.
```

Comment:
Similar to my comment above on property heat demand data from MCS - I'll make suggestions here based on the hard-coded data source that we can replace when we create a data getter. I'm also making a suggestion to add how an annual cost reduction is applied to the 2023 installation cost to estimate the installation cost for later years.

The installation cost data the spreadsheet model uses is from https://asf-heat-pump-affordability.s3.eu-west-2.amazonaws.com/2021_2023_ashp_installation_costs_deciles_2023GBP.xlsx

This data is generated from https://github.com/nestauk/asf_heat_pump_affordability/blob/dev/asf_heat_pump_affordability/notebooks/investigate_effect_of_room_number.py

```suggestion
                    Costs of installing air source heat pumps are estimated for the above housing archetypes and at different cost deciles. 

                    The air source heat pump installation costs for each property archetype and cost decile uses data from 2021 to Q2 2023 and is shown below. You can select the cost decile to view the corresponding installation costs in £ (adjusted for inflation and expressed in 2023 prices).         
```